docs/release-notes/2024-12-19-new-feature-available-for-anti-fraud-providers-gift-card-transaction-analysis.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "New feature available for anti-fraud providers: Gift card transaction analysis"
+slug: "2024-12-19-new-feature-available-for-anti-fraud-providers-gift-card-transaction-analysis"
+hidden: false
+type: "added"
+createdAt: "2024-12-19T00:00:00.219Z"
+updatedAt: "2024-12-19T00:00:00.219Z"
+excerpt: "Gift card transactions can now be analyzed with anti-fraud."
+---
+
+Anti-fraud providers integrated with VTEX can now analyze payment transactions made with [gift cards](https://help.vtex.com/en/tutorial/gift-card--tutorials_995).
+
+## What has changed?
+
+The `allowAntifraudOnGiftCard` field was created within the [Anti-fraud Provider Protocol](https://developers.vtex.com/docs/guides/how-the-integration-protocol-between-vtex-and-antifraud-companies-works), allowing anti-fraud providers to indicate whether they want to process gift card transaction data in a VTEX store.
+
+Gift card transaction information is sent to providers through the [Send anti-fraud pre-analysis data (optional)](https://developers.vtex.com/docs/api-reference/antifraud-provider-protocol#post-/pre-analysis) or [Send anti-fraud data](https://developers.vtex.com/docs/api-reference/antifraud-provider-protocol#post-/transactions) endpoints, as in payment operations made by credit or debit card.
+
+See an example of a payload sent on the [Send anti-fraud data](https://developers.vtex.com/docs/api-reference/antifraud-provider-protocol#post-/transactions) endpoint, showing the gift card information in the payments object.
+
+```json
+…
```

Comment:
Esses 3 pontos no início e fim são para indicar que existe mais informação no payload? Se sim, o que vc acha de retirar e dizer no texto que se trata de uma parte/trecho do payload?